torch/csrc/jit/runtime/register_distributed_ops.cpp

```diff
@@ -167,17 +199,16 @@ RegisterOperators reg_rpc_ops({
          },
          aliasAnalysisSpecialCase())});
 
-auto reg_distributed_ops =
-    torch::RegisterOperators()
-        .op("aten::get_gradients(int context_id) -> Dict(Tensor, Tensor)",
-            torch::RegisterOperators::options()
-                .aliasAnalysis(AliasAnalysisKind::FROM_SCHEMA)
-                .catchAllKernel([](int64_t context_id) {
-                  const auto& autogradContext =
-                      dist_autograd::DistAutogradContainer::getInstance()
-                          .retrieveContext(context_id);
-                  return autogradContext->getGradients();
-                }));
+auto reg_distributed_ops = torch::RegisterOperators().op(
+    "aten::get_gradients(int context_id) -> Dict(Tensor, Tensor)",
```

Comment:
Since this is not only worker info, does it make sense if we break this PR into two?

Reply:
Sure, I will put a formatting PR under.